README.md

```diff
@@ -88,6 +88,18 @@ Note that the default `service-id` includes more than the container name (see be
 
 	docker run -d --name redis-1 -e SERVICE_ID=redis-1 -p 6379:6379 redis
 
+#### Route53 backend
+
+AWS Route53 support for creating SRV DNS records. The URI provides an Route 53 hosted Zone Id. You can also enable the EC2 metadata service for hostname discovery using the `ec2hostname` parameter, otherwise the backend will default to the hostname provided by `os.Hostname()`. 
```

Comment:
The `ec2hostname` parameter only works for me if it's in all caps `EC2HOSTNAME`. Also, I think the option should be renamed, maybe `useEC2Metadata` or `useEC2MetadataForHostname` which is really long but more verbose. "EC2HOSTNAME" is a little to ambiguous though.